plugins/modules/clustering/pacemaker/pacemaker_cluster.py

```diff
@@ -0,0 +1,369 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2016, Mathieu Bultel <mbultel@redhat.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: pacemaker_cluster
+short_description: Manage pacemaker clusters
+author:
+- Mathieu Bultel (@mbultel)
+- Hani Audah (@haudah)
+description:
+   - This module can manage a pacemaker cluster and nodes from Ansible using
+     the pacemaker cli.
+options:
+    state:
+      description:
+        - Indicate desired state of the cluster
+      choices: [ cleanup, offline, online, restart, present, absent ]
+      type: str
+    node:
+      description:
+        - Specify which node of the cluster you want to manage. None == the
+          cluster status itself, 'all' == check the status of all nodes.
```

Comment:
It might make sense to split up cluster and node management into two modules, especially when there are operations that only apply to one of them.

In any case, I would rewrite the above as something like
```.yaml
  description:
    - Specify which node of the cluster you want to manage.
    - If not specified, the module will manage the cluster itself.
    - If the special value C(all) is used, all nodes will be managed.
```